sway-ir/src/parser.rs

```diff
@@ -121,15 +121,18 @@ mod ir_builder {
                     mdi
                 }
 
+            rule binary_op_kind() -> BinaryOpKind 
+                = "add" { BinaryOpKind::Add }
+                / "sub" { BinaryOpKind::Sub }
+                / "mul" { BinaryOpKind::Mul }
+                / "div" { BinaryOpKind::Div }
```

Comment:
Thanks, but another small nit -- by convention throughout this grammar the `_` is always placed right next to the literal strings.  I know this breaks my other complaint about reducing redundancy, but to keep it consistent could you put the `_` after each of these strings and take it out of `op_binary()` please?